Makefile

```diff
@@ -27,3 +28,11 @@ helm_dep_up:
 
 .PHONY: update_dependencies
 update_dependencies: setup_helm_repos helm_dep_up update_versions
+
+.PHONY: get_release_notes
+get_release_notes: setup_helm_repos helm_dep_up
+	git fecth --all --tags
```

Comment:
We do this so we ensure the local copy of the chart repository has all the tags necessary for calculating changelogs